cpplint.py

```diff
@@ -518,7 +520,27 @@ def u(x):
     xrange = range
     itervalues = dict.values
     iteritems = dict.items
+=======
+Py3k = (sys.version_info[0] == 3)
+"""A boolean to check if we are running Python3000"""
+
+try:
+    xrange(0,1)
+except NameError:
+    xrange = range
+try:
+    unicode
+except NameError:
+    basestring = unicode = str
+try:
+    long
+except NameError:
+    long = int
+>>>>>>> Added support for Python3000
+
```

Comment:
* My code introcudes a variable TEXT_TYPE and function u() to disambiguate between str and unicode, yours does `basestring = unicode = str`.
* I Use `range=xrange for py2, you use xrange=range for py3
* I redefine iteritems and itervalues functions, you dont
* you do `long = int`, I don't